std/meta.d

```diff
@@ -1188,6 +1169,21 @@ template aliasSeqOf(alias range)
     }
 }
 
+@safe unittest
```

Comment:
It could stand to have a few more unittests testing different types of iterables.

Reply:
There are already quite a few existing that cover the range types, did have anything specific in mind?

Reply:
Ok, infinite ranges were allowed with the previous implementation as well but obviously that's very bad. Done